drivers/java/driver.go

```diff
@@ -274,18 +276,24 @@ func (d *Driver) RecoverTask(handle *drivers.TaskHandle) error {
 	execImpl, pluginClient, err := executor.ReattachToExecutor(plugRC,
 		d.logger.With("task_name", handle.Config.Name, "alloc_id", handle.Config.AllocID))
 	if err != nil {
-		d.logger.Error("failed to reattach to executor", "error", err, "task_id", handle.Config.ID)
+		d.logger.Error("failed to reattach to executor, maybe died", "error", err, "task_id", handle.Config.ID)
```

Comment:
`task_id` doesn't mean much to the user, probably safe to remove from error logging

Reply:
Agree - but I'm matching the log patterns in this function.  Given that this PR is a bit large, I'll address it in a subsequent PR.